sdks/python/build.gradle

```diff
@@ -327,24 +327,35 @@ task hdfsIntegrationTest(dependsOn: 'installGcpTest') {
   }
 }
 
+class CompatibilityMatrixConfig {
+  String type
```

Comment:
Add a comment, that this can be Batch or Streaming? Or convert this to boolean `isStreaming`.

Reply:
Will make it a boolean